### ISSUE TITLE
Accessibility issue with MultiDopdownList

### ISSUE THREAD

**Issue by allendol** [appbaseio/reactivesearch#1115]
I need to use the tab key to go to MultiDopdownList. In MultiDopdownList I select two or more options using the keyboard and not mouse.
The I leave the field (component). But the drop down still stays open.
There should be a way that the opened MultiDopdownList to be collapsed or closed without using the mouse.
![MultiDropDownList](https://user-images.githubusercontent.com/40170767/63592600-f301a300-c57f-11e9-964d-7440029bf3b4.png)
There is a problem with accessibility in MultiDopdownList unless I am missing something. 
Any workaround?

**Comment by jyash97** [comment]
Hey @allendol  
Thanks for reporting the bug. I was able to replicate this I think we can use `escape` key to close the dropdown.

**Comment by allendol** [comment]
Strange, the escape key doesn't work for me. I tried earlier. Which browser are you using? I use Edge.

**Comment by jyash97** [comment]
Oh sorry, I didn't phrase it well.  Currently, it is not closing the dropdown using the escape key.  But I can check on this to close the dropdown when the escape key is pressed. Would this be a good solution for this ?

**Comment by allendol** [comment]
Yes, we definitely need a solution for it. Escape key can be a solution.

**Comment by allendol** [comment]
@jyash97 Do you know when the fixes will be available?

**Comment by jyash97** [comment]
Hey @allendol 
Have created a fix for this in PR #1126 .

**Comment by allendol** [comment]
Hi @jyash97 
Thanks for the quick workaround. I am trying to add the changes to Download.JS located at \node_modules\@appbaseio\reactivesearch\lib\components\shared\Download.JS but looks like this JS file is not called at all.
Do I miss anything?

**Comment by allendol** [comment]
I found my problem. The location was correct. I had to run NPM start again. It is working.
Thanks again!

**Comment by jyash97** [comment]
Hey @allendol 
Right after doing the changes on `node_modules` you will need to re-run your app to see the changes.
